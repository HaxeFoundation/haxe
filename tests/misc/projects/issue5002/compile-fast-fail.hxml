compile-fail.hxml
-D fail-fast
