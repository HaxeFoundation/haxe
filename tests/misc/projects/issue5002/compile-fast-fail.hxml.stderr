Main.hx:1: characters 1-8 : "0" is not a valid field name.
